src/components/ChatLog.jsx

```diff
@@ -0,0 +1,30 @@
+import PropTypes from 'prop-types';
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+
+const ChatLog = ({ entries, onLikeToggle }) => {
+  const localSender = 'Vladimir';
```

Comment:
This is a constant variable so we should use all capital letters to name it:
```suggestion
  const LOCAL_SENDER = 'Vladimir';
```